src/3d/framegraph/qgspostprocessingrenderview.cpp

```diff
@@ -0,0 +1,193 @@
+  colorOutput->setAttachmentPoint( Qt3DRender::QRenderTargetOutput::Color0 );
+
+  // Create a texture to render into.
+  mRenderCaptureColorTexture = new Qt3DRender::QTexture2D( colorOutput );
+  mRenderCaptureColorTexture->setSize( mSize.width(), mSize.height() );
+  mRenderCaptureColorTexture->setFormat( Qt3DRender::QAbstractTexture::RGB8_UNorm );
+  mRenderCaptureColorTexture->setMinificationFilter( Qt3DRender::QAbstractTexture::Linear );
+  mRenderCaptureColorTexture->setMagnificationFilter( Qt3DRender::QAbstractTexture::Linear );
+  mRenderCaptureColorTexture->setObjectName( mViewName + "::ColorTarget" );
+
+  // Hook the texture up to our output, and the output up to this object.
+  colorOutput->setTexture( mRenderCaptureColorTexture );
+  renderTarget->addOutput( colorOutput );
+
+  Qt3DRender::QRenderTargetOutput *depthOutput = new Qt3DRender::QRenderTargetOutput( renderTarget );
+
+  depthOutput->setAttachmentPoint( Qt3DRender::QRenderTargetOutput::Depth );
+  mRenderCaptureDepthTexture = new Qt3DRender::QTexture2D( depthOutput );
+  mRenderCaptureDepthTexture->setSize( mSize.width(), mSize.height() );
+  mRenderCaptureDepthTexture->setFormat( Qt3DRender::QAbstractTexture::DepthFormat );
+  mRenderCaptureDepthTexture->setMinificationFilter( Qt3DRender::QAbstractTexture::Linear );
+  mRenderCaptureDepthTexture->setMagnificationFilter( Qt3DRender::QAbstractTexture::Linear );
+  mRenderCaptureDepthTexture->setComparisonFunction( Qt3DRender::QAbstractTexture::CompareLessEqual );
+  mRenderCaptureDepthTexture->setComparisonMode( Qt3DRender::QAbstractTexture::CompareRefToTexture );
+  mRenderCaptureDepthTexture->setObjectName( mViewName + "::DepthTarget" );
+
+  depthOutput->setTexture( mRenderCaptureDepthTexture );
+  renderTarget->addOutput( depthOutput );
+
+  mRenderCaptureTargetSelector->setTarget( renderTarget );
+  // =============== ^ NEED ONLY DURING UNIT TEST ^
+
+  mSubPassesNode = new Qt3DRender::QFrameGraphNode( mRenderCaptureTargetSelector );
+  mSubPassesNode->setObjectName( mViewName + "::Sub passes top node" );
+
+  return mRenderCaptureTargetSelector;
+}
+
+
+Qt3DRender::QFrameGraphNode *QgsPostprocessingRenderView::constructSubPostPassForProcessing( QgsShadowRenderView &shadowRenderView,       //
+                                                                                             QgsForwardRenderView &forwardRenderView,     //
+                                                                                             QgsAmbientOcclusionRenderView &aoRenderView, //
+                                                                                             Qt3DCore::QEntity *rootSceneEntity )
+{
+  Qt3DRender::QCameraSelector *cameraSelector = new Qt3DRender::QCameraSelector;
+  cameraSelector->setObjectName( mViewName + "::Sub pass::Postprocessing" );
+  cameraSelector->setCamera( shadowRenderView.lightCamera() );
```

Comment:
it should be safe to remove QCameraSelector, as the post processing entity does not use any camera - it's just a full-screen quad...